R/module_nested_tabs.R

```diff
@@ -112,8 +112,7 @@ ui_nested_tabs.teal_module <- function(id, modules, datasets, depth = 0L, is_mod
   checkmate::assert_class(datasets, class = "FilteredData")
   ns <- NS(id)
 
-  args <- isolate(teal.transform::resolve_delayed(modules$ui_args, datasets))
```

Comment:
are you sure?
if yes - is this the only place where we use this function?
if yes - shall we remove it?

Reply:
Yup, too early for this. I was testing something and pushed by mistake. Reverting.

FYI we will remove `resolve_delay` from `ui` functions anyway soon.